### PR TITLE
Portfolio: add a contact form and finish the mobile version

### PR DESCRIPTION
**Pull request by udeaghad**
In this pull request,
- We built a contact form in the footer section
- created the name, e-mail, and message field
- ensured that all these fields had placeholder
- also ensured that these fields are required and cannot be submitted if any of it is empty
- collected data on https://formspree.io/forms/mrgjoqgg/submissions
- used flexbox to layout the elements
- and also implemented the button interactions.

Any other suggestions on this project will be appreciated.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -435,7 +435,70 @@ <h4 class="languages">Skills</h4>
...
+                    <li>
+                    
+                     <textarea id="more-info" name="more-info" placeholder="Enter your text here" maxlength="500" required></textarea>
+                    </li>
+                </ul>
+
+                <button class="button resume project-button" type="submit" >Get in touch</button>
+                
+            
+            </form>
```
Comment (CollinsTatang) [inline index.html:462]: - [ ] Hi, a nice implementation of the contact form and design but your form is not responsive to mobile view.
Kindly make your form responsive to all screen size
matchres
.

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (udeaghad) [reply index.html:462]: The webpage has been optimized to become responsive.

**Review by BenMukebo — APPROVED** [review]
<h1>Status: Approve  :champagne:  :+1: :confetti_ball: 

### **Hi @udeaghad**, 

### **Congratulations! 🎉 Project Approved ✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 
- The form is submitted successfully with valid input values. :heavy_check_mark:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [ OPTIONAL] This is a group project. Please let's include all the authors in the README file.
- [ OPTIONAL]  Please let's make sure that contribution to a group project is distributed equally among the members( commit messages should be fairly equal).

Cheers :clinking_glasses:  and Happy coding!!! :dancers:💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -435,7 +435,70 @@ <h4 class="languages">Skills</h4>
...
+                    <li>
+                       <input type="email" id="email" placeholder="E-mail address" name="e-mail" required>
+                    </li>
+                    <li>
+                    
+                     <textarea id="more-info" name="more-info" placeholder="Enter your text here" maxlength="500" required></textarea>
+                    </li>
+                </ul>
+
+                <button class="button resume project-button" type="submit" >Get in touch</button>
```
Comment (BenMukebo) [inline index.html:459]: - [ OPTIONAL] The Button interactions such as hover, active (when it's pressed), are styled as shown in the Figma template. Please make this ⬇️ 


top1

Comment (udeaghad) [reply index.html:459]: The correct button was actually implemented as on the Figma template.

**Pull request merged**